index.js

```diff
@@ -205,6 +205,10 @@ function loadConfig(program) {
         saveMessages.push(`Tags: ${processed.diff} been saved to config.`)
       }
 
+      if (program.sendUserAgent || process.env.LOGGER_SEND_USER_AGENT) {
+        parsedConfig.sendUserAgent = program.sendUserAgent || process.env.LOGGER_SEND_USER_AGENT
```

Comment:
@darinspivey I see [the default value](https://github.com/logdna/logger-node/blob/master/lib/logger.js#L79) for `sendUserAgent` within `logger-node` is `true` - I'd suggest to use it here too to make the default / fallback value visible - overall, I'd say instead of `if` block, my suggestion would be just
```
parsedConfig.sendUserAgent = program.sendUserAgent || process.env.LOGGER_SEND_USER_AGENT || true
```

Reply:
Hmm, not sure I agree.  Doing this would set a default for the default.  I feel like that should be handled by the logger, therefore don't pass anything if it's not defined.

Reply:
I can remove the `if` block though, if that works for you?